site/docs/v1/tech/account-management/customizing-account-management.adoc

```diff
@@ -0,0 +1,96 @@
+---
+layout: doc
+title: Account Management - Customizing
+description: Managing Accounts - Customizing
+---
+:page-liquid:
+
+include::docs/v1/tech/shared/_premium-edition-blurb.adoc[]
+
+[NOTE.since]
+====
+Available since 1.26.0
+====
+
+== Customizing Account Management
+
+FusionAuth ships with a default template for the Account Management page. However, you are not limited to the default form fields or the theme for that matter. Below we will walk you through updating both.
+
+=== Add a Form Field
+
+[NOTE]
+====
+Forms allow you to collect any data you want. More information can be found in our link:/docs/v1/tech/guides/advanced-registration-forms/[Advanced Registration Forms] page.
+====
+
+Below we are going to extend the usefulness of our account management page by adding a field to the `User Self Service Form`.
+
+To change a form:
+
+1. Navigate to `icon:check-square[type=fas] Forms`
+2. Click the `icon:edit[type=fas ,role=ui-button blue] Edit` on `Default User Self Service provided by FusionAuth`.
+
+image::account-management/forms-home-self-serve-crop.png[Form Home, width=1200, role=bottom-cropped]
+
+==== Edit Form
+
+In this example, we are going to add a `birthDate` field on our account management self service form.
+
+image::account-management/edit-default-account-form.png[Form Home, width=1200, role=bottom-cropped]
+
+Remember, you can move each field around with `icon:arrow-up[type=fas] up` and `icon:arrow-down[type=fas] down` arrows
+
+==== Adding Birthday
+
+To add a `birthDate`, click `icon:plus[type=fas] Add field`
+
+image::account-management/add-field-popup-birthdate.png[Form Home, width=600, role=bottom-cropped top-cropped box-shadow]
+
+Select `Birthdate` and `icon:save[type=fas] Submit`. Now we have a custom form available on our self-service account management page.
+
+Let's take a look. Navigate to --
+
+- Your account management page.
+- Login as a user of this application.
+- Click on the `icon:pencil[type=fas] Edit` icon in the top right.
+
+image::account-management/account-form-with-birthday-crop.png[Form Edits with Birthday, width=1200, role=bottom-cropped]
+
+icon:trophy[] Success! Now we have a user with an editable `birthDate` field.
+
+This is a simple example, but with FusionAuth forms, you can extend this account management form to whatever your business requirements demand.
```

Comment:
@mooreds 

Not sure if this is what you were going for but revised as such
```
This is a simple example, but with FusionAuth forms, you can extend this account management form to meet most business requirements.

With FusionAuth forms you can achieve more complex workflows such as:

- Creating multi-step registration forms (for instance) by adding `sections`.
- Specifying if a field is required or optional.
- Making certain forms editable by an admin user, but not by a 'regular' user.

Below, we will show you how to make this user-editable `birthDate` field show up on the account management home screen.
```

Reply:
That's great! Did you confirm this: `Making certain forms editable by an admin user, but not by a 'regular' user`? I don't know if that is possible, but it has been a while since I looked at that functionality.

Reply:
Can I have a form field that is editable by an admin but not by a user? = 👎🏻 
Making certain forms editable by an admin user, but not by a 'regular' user = 👍🏻  

You can have an admin form and a user form from what I can tell.